CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+project(TCP-Server)
+
+cmake_minimum_required(VERSION 3.5)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS "-fcoroutines")
+
+find_package (Valgrind)
+find_package (Seastar REQUIRED)
+
+# Build interface used by Rust.
+add_library(rust_ffi STATIC
+    server/server.cc
+    server/rust_ffi/rust_ffi.cc
+)
```

Comment:
I don't get the purpose of this target at all - why is it necessary, it looks like it could be removed. Can you explain it to me?